modules/tiles/src/tileset/traversers/tileset-traverser.js

```diff
@@ -43,15 +43,8 @@ export default class TilesetTraverser {
     // update tile (visibility and expiration)
     this.updateTile(root, frameState);
 
-    // check if traversable
-    if (!this.canTraverse(root, frameState, true)) {
```

Comment:
after this removed, whether `root` tile should be selected for rendering will be checked in `executeTraversal`